das/daser.go

```diff
@@ -17,25 +18,39 @@ import (
 
 var log = logging.Logger("das")
 
-// DASer continuously validates availability of data committed to headers.
-type DASer struct {
-	ctx    context.Context
-	cancel context.CancelFunc
+const (
+	priorityBufferSize = 1024
+	concurrency        = 256
```

Comment:
I really like the process how you measured and choose this value. Can you pls document it?